src/one_dragon/base/operation/application/plugin_import_service.py

```
@@ -0,0 +1,494 @@
+
+    def delete_plugin(self, plugin_dir: str | Path) -> ImportResult:
+        """删除插件
+
+        Args:
+            plugin_dir: 插件目录路径或名称
+
+        Returns:
+            ImportResult: 删除结果
+        """
+        if isinstance(plugin_dir, str):
+            # 如果只是目录名，构建完整路径
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)